### ISSUE TITLE
Loadable nodes not extending interfaces other than node

### ISSUE THREAD

**Issue by NICHTJ3** [hayes/giraphql#88]
The below code sample produces the second when I believe it should work the same as the non loadable version

```js
export const UserEntity = builder.loadableNode('UserNode', {
  id: {
    resolve: (user) => user.id,
  },
  interfaces: [PageInterface],
  // ...
  fields: (t) => ({
    name: t.exposeString('name', {}),
   // ...
    posts: t.connection(
      {
        type: PostEntity,
        authScopes: {
          authenticated: true,
        },
        resolve: async ({ id }, args) =>
          // NOTE: This seems ineficient
          resolveOffsetConnection({ args }, async ({ limit, offset }) =>
            prisma.post.findMany({
              where: {
                creatorID: id,
              },
              include: {
                creator: true,
              },
              take: limit ?? undefined,
              skip: offset ?? undefined,
            }),
          ),
      },
      {},
      {},
    ),
  }),
});

```

```graphql
type UserNode implements Node {
  avatarUrl: String!
  bio: String
  createdAt: DateTime!
  email: String!
  followRequests: [FollowRequest!]!
  followStatus: FriendShipStatus!
  followers: [UserNode!]!
  following: [UserNode!]!
  id: ID!
  linkedSite: String
  name: String!
  postCount: Int!
  posts(after: ID, before: ID, first: Int, last: Int): UserNodePostsConnection!
  username: String!
}
```

**Comment by NICHTJ3** [comment]
It appears the interfaces passed to `loadableNode` are never used:
- https://github.com/hayes/giraphql/blob/0fe0b0b5cb5507a9460bb4d23ca2520303f91c87/packages/deno/packages/plugin-dataloader/schema-builder.ts#L29

**Comment by hayes** [comment]
thanks! fixed in @giraphql/plugin-dataloader@2.5.1
